website/components/features-list/style.module.css

```diff
@@ -0,0 +1,107 @@
+.featureListContainer {
+  background-color: #000;
+  padding-top: 128px;
+  background-position: right top, left bottom;
+  background-repeat: no-repeat;
+  color: #fff;
+}
+
+.featureContainer {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+
+  & .featureTextContainer {
```

Comment:
Probably some margin-right on desktop here or something to ensure some breathing room at very specific breakpoints.

These but up against each-other.

<img width="1090" alt="CleanShot 2021-09-21 at 11 10 31@2x" src="https://user-images.githubusercontent.com/2105067/134224742-94bc0ec2-9bf3-4c15-a63d-2fe46cb7bb02.png">